--- chrome/browser/ui/views/webid/account_selection_bubble_view.cc.orig	2022-04-22 22:14:55.104993200 -0500
+++ chrome/browser/ui/views/webid/account_selection_bubble_view.cc	2022-04-22 22:10:20.892306500 -0500
@@ -267,7 +267,7 @@
   // being picked.
   RemoveAllChildViews();
   AddChildView(std::make_unique<views::Separator>());
-  std::vector<const content::IdentityRequestAccount> accounts = {account};
+  std::vector<content::IdentityRequestAccount> accounts = {account};
   AddChildView(CreateAccountChooser(accounts));
   SizeToContents();
   PreferredSizeChanged();
