--- a/sql/recover_module/cursor.h
+++ b/sql/recover_module/cursor.h
@@ -62,8 +62,8 @@ class VirtualCursor {
   // |sqlite_cursor| must have been returned by VirtualTable::SqliteCursor().
   static inline VirtualCursor* FromSqliteCursor(
       sqlite3_vtab_cursor* sqlite_cursor) {
-    static_assert(std::is_standard_layout<VirtualCursor>::value,
-                  "needed for the reinterpret_cast below");
+    /*static_assert(std::is_standard_layout<VirtualCursor>::value,
+                  "needed for the reinterpret_cast below");*/
     static_assert(offsetof(VirtualCursor, sqlite_cursor_) == 0,
                   "sqlite_cursor_ must be the first member of the class");
     VirtualCursor* result = reinterpret_cast<VirtualCursor*>(sqlite_cursor);
