From f2093699969cb81a3eac52320ac3d755d4049a6d Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 26 Apr 2022 12:36:31 +0200
Subject: [PATCH] IWYU: add limits for std::max in printing::PageRange

Bug: 957519
Change-Id: I52b108e48d723e969f66df81a9aa478a1e331b31
---

diff --git a/printing/page_range.h b/printing/page_range.h
index 07414b3..218e758 100644
--- a/printing/page_range.h
+++ b/printing/page_range.h
@@ -7,6 +7,7 @@
 
 #include <stdint.h>
 
+#include <limits>
 #include <vector>
 
 #include "base/component_export.h"
