--- third_party/blink/renderer/core/html/track/vtt/vtt_cue_layout_algorithm.cc.orig	2022-07-16 19:15:23.489060700 -0500
+++ third_party/blink/renderer/core/html/track/vtt/vtt_cue_layout_algorithm.cc	2022-07-16 19:13:47.051902400 -0500
@@ -39,7 +39,7 @@
 
   // 4. Let line be cue's computed line.
   // 5. Round line to an integer by adding 0.5 and then flooring it.
-  float line = std::floorf(snap_to_lines_position_ + 0.5f);
+  float line = floorf(snap_to_lines_position_ + 0.5f);
 
   // 6. Vertical Growing Left: Add one to line then negate it.
   const auto& cue_box = *cue_.GetLayoutBox();
