--- chrome/browser/chrome_browser_main_extra_parts_linux.cc.orig	2022-05-19 14:02:09.713777400 -0500
+++ chrome/browser/chrome_browser_main_extra_parts_linux.cc	2022-05-19 14:01:15.117444600 -0500
@@ -10,8 +10,8 @@
 #include "ui/ozone/buildflags.h"
 #include "ui/ozone/public/ozone_switches.h"
 
-#if BUILDFLAG(OZONE_PLATFORM_WAYLAND)
 #include "base/environment.h"
+#if BUILDFLAG(OZONE_PLATFORM_WAYLAND)
 #include "base/files/file_path.h"
 #include "base/files/file_util.h"
 #include "base/nix/xdg_util.h"
