From 0e727e2340f355b3f17d83338986ca47fe7a58fd Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 25 Apr 2022 10:47:55 +0200
Subject: [PATCH] IWYU: add utility for std::exchange

Bug: 957519
Change-Id: I307d520fdc34d2452018ed32a505e7e519739410
---

diff --git a/base/third_party/symbolize/symbolize.h b/base/third_party/symbolize/symbolize.h
index 99029a9..278078f 100644
--- a/base/third_party/symbolize/symbolize.h
+++ b/base/third_party/symbolize/symbolize.h
@@ -61,6 +61,7 @@
 #ifdef HAVE_SYMBOLIZE
 
 #include <algorithm>
+#include <utility>
 
 #if defined(__ELF__)  // defined by gcc
 #if defined(__OpenBSD__)
