--- src/chromium-109.0.5402.0/content/public/browser/content_browser_client.cc.orig	2022-11-06 08:14:16.377814700 -0600
+++ src/chromium-109.0.5402.0/content/public/browser/content_browser_client.cc	2022-11-05 21:46:10.161979400 -0500
@@ -19,6 +19,7 @@
 #include "base/values.h"
 #include "build/build_config.h"
 #include "build/chromeos_buildflags.h"
+#include "content/public/browser/anchor_element_preconnect_delegate.h"
 #include "content/public/browser/authenticator_request_client_delegate.h"
 #include "content/public/browser/browser_accessibility_state.h"
 #include "content/public/browser/browser_main_parts.h"
