--- a/base/check_op.cc
+++ b/base/check_op.cc
@@ -13,6 +13,7 @@
 
 #include <cstdio>
 #include <sstream>
+#include <string.h>
 
 namespace logging {
 
--- a/base/synchronization/lock_impl.h
+++ b/base/synchronization/lock_impl.h
@@ -5,6 +5,8 @@
 #ifndef BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 #define BASE_SYNCHRONIZATION_LOCK_IMPL_H_
 
+#include <cstring>
+
 #include "base/base_export.h"
 #include "base/logging.h"
 #include "base/macros.h"
--- a/base/trace_event/heap_profiler_allocation_context_tracker.cc
+++ b/base/trace_event/heap_profiler_allocation_context_tracker.cc
@@ -6,6 +6,7 @@
 
 #include <algorithm>
 #include <iterator>
+#include <string.h>
 
 #include "base/atomicops.h"
 #include "base/debug/debugging_buildflags.h"
