--- chrome/browser/ui/views/autofill_assistant/password_change/assistant_side_panel_coordinator_impl.h.orig	2022-05-18 10:03:56.661882900 -0500
+++ chrome/browser/ui/views/autofill_assistant/password_change/assistant_side_panel_coordinator_impl.h	2022-05-18 10:04:09.828158700 -0500
@@ -5,6 +5,8 @@
 #ifndef CHROME_BROWSER_UI_VIEWS_AUTOFILL_ASSISTANT_PASSWORD_CHANGE_ASSISTANT_SIDE_PANEL_COORDINATOR_IMPL_H_
 #define CHROME_BROWSER_UI_VIEWS_AUTOFILL_ASSISTANT_PASSWORD_CHANGE_ASSISTANT_SIDE_PANEL_COORDINATOR_IMPL_H_
 
+#include <memory>
+
 #include "base/memory/raw_ptr.h"
 #include "chrome/browser/ui/autofill_assistant/password_change/assistant_side_panel_coordinator.h"
 #include "chrome/browser/ui/views/side_panel/side_panel_entry_observer.h"
