--- third_party/perfetto/src/trace_processor/containers/bit_vector.cc.orig	2022-06-15 15:00:00.139178700 -0500
+++ third_party/perfetto/src/trace_processor/containers/bit_vector.cc	2022-06-15 15:00:23.024788400 -0500
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#include <limits>
 #include "src/trace_processor/containers/bit_vector.h"
 
 #include "src/trace_processor/containers/bit_vector_iterators.h"
