--- a/third_party/blink/renderer/platform/bindings/v8_set_return_value.h
+++ b/third_party/blink/renderer/platform/bindings/v8_set_return_value.h
@@ -51,7 +51,7 @@ void V8SetReturnValue(const CallbackInfo
 
 // nullptr
 template <typename CallbackInfo>
-void V8SetReturnValue(const CallbackInfo& info, nullptr_t) {
+void V8SetReturnValue(const CallbackInfo& info, std::nullptr_t) {
   info.GetReturnValue().SetNull();
 }
 
