--- src/chromium-107.0.5277.0/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc.orig	2022-09-02 21:26:58.812451500 -0500
+++ src/chromium-107.0.5277.0/third_party/tflite_support/src/tensorflow_lite_support/metadata/cc/utils/zip_readonly_mem_file.cc	2022-09-02 21:27:17.619689200 -0500
@@ -19,7 +19,7 @@
 #include <cstdio>
 
 #include "absl/strings/string_view.h"  // from @com_google_absl
-#include "contrib/minizip/ioapi.h"
+#include "third_party/zlib/contrib/minizip/ioapi.h"
 
 namespace tflite {
 namespace metadata {
