--- chrome/browser/ui/views/webid/account_selection_bubble_view.h.orig	2022-06-22 14:55:16.286083600 -0500
+++ chrome/browser/ui/views/webid/account_selection_bubble_view.h	2022-06-22 11:55:04.258346300 -0500
@@ -128,7 +128,7 @@
 
   // The list of accounts to select from. Not updated when the user selects an
   // account and navigates to the privacy policy / terms of service page.
-  const std::vector<const content::IdentityRequestAccount> account_list_;
+  const std::vector<content::IdentityRequestAccount> account_list_;
 
   // The TabStripModel of the current browser. We need this in order to show the
   // privacy policy and terms of service urls when the user clicks on the links.
