--- third_party/blink/renderer/platform/wtf/text/ascii_fast_path.h.orig	2022-06-24 21:43:26.549472800 -0500
+++ third_party/blink/renderer/platform/wtf/text/ascii_fast_path.h	2022-06-24 21:43:47.639844000 -0500
@@ -22,6 +22,7 @@
 #ifndef THIRD_PARTY_BLINK_RENDERER_PLATFORM_WTF_TEXT_ASCII_FAST_PATH_H_
 #define THIRD_PARTY_BLINK_RENDERER_PLATFORM_WTF_TEXT_ASCII_FAST_PATH_H_
 
+#include <limits>
 #include <stdint.h>
 
 #include "base/check_op.h"
