--- a/chrome/browser/enterprise_reporting/notification/extension_request_notification.cc
+++ b/chrome/browser/enterprise_reporting/notification/extension_request_notification.cc
@@ -46,7 +46,7 @@ constexpr int kNotificationBodies[] = {
 
 }  // namespace
 
-ExtensionRequestNotification::ExtensionRequestNotification() = default;
+ExtensionRequestNotification::ExtensionRequestNotification() {}
 
 ExtensionRequestNotification::ExtensionRequestNotification(
     Profile* profile,
