--- a/third_party/angle/BUILD.gn
+++ b/third_party/angle/BUILD.gn
@@ -458,23 +458,9 @@ config("commit_id_config") {
 }
 
 commit_id_output_file = "$root_gen_dir/angle/commit.h"
-action("commit_id") {
-  script = "src/commit_id.py"
+copy("commit_id") {
+  sources = [ "src/commit.h" ]
   outputs = [ commit_id_output_file ]
-
-  # Add git as a dependency if it is available.
-  angle_git_is_present =
-      exec_script("src/commit_id.py", [ "check" ], "value") == 1
-  if (angle_git_is_present) {
-    # commit id should depend on angle's HEAD revision
-    inputs = [ ".git/HEAD" ]
-  }
-
-  args = [
-    "gen",
-    rebase_path(commit_id_output_file, root_build_dir),
-  ]
-
   public_configs = [ ":commit_id_config" ]
 }
 
